_posts/2020-02-15-pmemkv-bindings.md

```diff
@@ -0,0 +1,155 @@
+either translating each status into numeric value or wrapping them into exceptions. Bindings
+using exceptions do not use status "OK", just returns nothing or a numeric value
+(of `PMEMKV_STATUS_OK`) if no error occurred. Return value differs also for various functions -
+to see details read binding's documentation or pmemkv's man pages.
+
+#### Not always direct access to data
+
+**Except in Python** (and soon [in Node.js][nodejs_pr]), our bindings do not (yet)
+provide direct access to the data stored in the Persistent Memory.
+They need to use buffer to copy the data to.
+
+#### Examples, tests and CI
+
+Each repository is equipped with basic, needed features like:
+* examples, showing basic library usage,
+* tests, verifying current implementation,
+* Continuous Integration system, to check all incoming changes.
+
+#### Packaging specific for language
+
+Each language is different, so we try our best to use packaging and installation systems
+native to (or one of the most common for) each binding's language.
+
+#### Contributions are welcome
+
+As all of pmem's organization repositories, the ones of pmemkv-bindings are also
+open source and in each of them any support is more than welcome.
+
+### pmemkv-nodejs
+
+Node.js binding is the first one to be released in stable version (1.0). It means
+it guarantees backward compatibility and it provides API functionally equal to
+the native pmemkv C/C++ API **in version 1.0**.
+
+It also delivers API description in form of JSDoc documentation (accessible e.g. as html in
+[here](nodejs_pmem_io)) and NodeJS-native support for configuring engines by JSON object.
+It handles returned statuses as exceptions and does not return any value when no errors occur.
+Installation of this project is supported by simply typing `npm install` within the source directory.
+It then installs the dependencies listed in package.json in the local node_modules folder.
+
+For more information, like required dependencies, usage and full installation guide,
+see [pmemkv-nodejs github repository][nodejs_gh].
+
+### pmemkv-python
+
+This is the second binding to be released in version 1.0. It has all functionalities
+from pmemkv 1.0.
+
+It also delivers API description in form of PyDoc documentation (accessible e.g. as html in
+[here](py_pmem_io)) and Python-native data type (`dict`) to store engine's configuration items.
+It handles returned statuses as exceptions and does not return any value when no errors occur.
+To install this project in your system it's required to type in: `python3 setup.py install [--user]`,
+and then it is installed in a system path or (when optional parameter `--user` is given) within
+user's files (`~/.local/lib/`).
+
+For more information, like required dependencies, usage and full installation guide,
+see [pmemkv-python github repository][py_gh].
+
+### pmemkv-java
+
+Latest released version is 0.9 and it's working just fine with libpmemkv 1.0, but
```

Comment:
and is compatible with libpmemkv 1.0, but comes with no guarantees of stable API.